tests/test_core.py

```diff
@@ -5629,14 +5629,20 @@ def test_dlmalloc_inline(self):
 
   @no_asan('depends on the specifics of memory size, which for asan we are forced to increase')
   def test_dlmalloc(self):
-    self.banned_js_engines = [config.NODE_JS] # slower, and fail on 64-bit
+#    self.banned_js_engines = [config.NODE_JS] # slower, and fail on 64-bit
```

Comment:
this looks unintentional? Or is this line out of date and it should be removed?

Reply:
Err, yeah, I seem to be on a spree of leaking test changes to commits. Cleaned up now.